utils/luts.ts

```diff
@@ -0,0 +1,19 @@
+export const VALID_COMMUNITIES = ["AK91", "AK182"] as const;
```

Comment:
Would it make more sense to make CommunityId a Type?  Then all this enforcement kinda just happens.